### ISSUE TITLE
error at 2.1.2 with contao.security.token_checker Service

### ISSUE THREAD

**Issue by zonky2** [MetaModels/attribute_checkbox#24]
error at 2.1.2 with contao.security.token_checker service

```The service "metamodels.attribute_checkbox.published_filtersetting.factory"                                                                           
     has a dependency on a non-existent service "contao.security.token_checker" 
```
this service is available from contao 4.5.0

see https://community.contao.org/de/showthread.php?77583-Fehler-beim-Update

**Comment by zonky2** [comment]
@richardhj update unittest pls

**Comment by stefanheimes** [comment]
Fixed with version 2.1.3
